frb_example/pure_dart/rust/Cargo.toml

```diff
@@ -8,12 +8,15 @@ name = "flutter_rust_bridge_example"
 crate-type = ["cdylib"]
 
 [build-dependencies]
-flutter_rust_bridge_codegen = { path = "../../../frb_codegen" }
+flutter_rust_bridge_codegen = { path = "../../../frb_codegen", features = [
+  "chrono",
```

Comment:
I guess it is the default feature so we do not need this "features=..." modification any more?

Reply:
resolved in 42442f7f66857730980abe373d6b77c05d778710